database/src/main/resources/changesets/workflow_store.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <property name="clob.type" value="LONGTEXT" dbms="mysql"/>
+    <property name="clob.type" value="LONGVARCHAR" dbms="hsqldb"/>
+
+    <changeSet author="chrisl" id="WORKFLOW_STORE">
+        <comment>
+            Temporary storage area for workflows that are still in progress.
+        </comment>
+        <createTable tableName="WORKFLOW_STORE">
+            <column autoIncrement="true" name="WORKFLOW_STORE_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_WORKFLOW_STORE"/>
+            </column>
+            <column name="WORKFLOW_UUID" type="VARCHAR(255)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="WORKFLOW_DEFINITION" type="${clob.type}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WORKFLOW_INPUTS" type="${clob.type}">
+                <constraints nullable="true"/>
+            </column>
+            <column name="WORKFLOW_OPTIONS" type="${clob.type}">
+                <constraints nullable="true"/>
+            </column>
+            <column name="STATE" type="varchar(15)" defaultValue="Submitted">
```

Comment:
I might suggest `WORKFLOW_STATE` both for consistency and to avoid running into a reserved word for some RDBMS implementation.